docs/customization/repository.rst

```diff
@@ -18,7 +18,7 @@ You may need for instance:
 How to customize a Repository?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let's assume that you would want to find products that are the bestsellers in your shop.
+Let's assume that you would want to find products that you are running out of in the inventory.
 
 1. Create your own repository class under the ``AppBundle\Repository`` namespace.
```

Comment:
No, just mentioning all the time, that we have such a feature ;) Even we are forgetting about it all the time.